make-coe/make-coe.go

```diff
@@ -180,7 +180,26 @@ func (carina *MakeCOE) DeleteCluster(token string) (common.Cluster, error) {
 
 // GrowCluster adds nodes to a cluster by its id or name (if unique)
 func (carina *MakeCOE) GrowCluster(token string, nodes int) (common.Cluster, error) {
-	return nil, errors.New("[make-coe] Growing clusters from the carina cli is not supported yet")
+	return nil, errors.New("[make-coe] Grow command not supported. Please use 'resize'.")
+}
+
+// ResizeCluster adds nodes to a cluster
```

Comment:
Let's have this function doc match the text for the interface it's implementing `ResizeCluster resizes the cluster to the specified number of nodes`.